package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "webpack-practice",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack serve"
```

Comment:
- [OPTIONAL] I highly recommend you change this script name to `start` which is the most popular one to start the server. It will be easier for any other folks who want to test your project locally.

Reply:
hi @levy002 I hope you are doing well sir

I cannot understand which change I should make like replacing scripts to start.
or main to start?

Reply:
@PowerLevel9000, You can rename your script `start` instead of`dev`, so to start your server we will run `npm run start`.

Reply:
@levy002  thank you sir